pypit/arcoadd.py

```diff
@@ -500,8 +500,19 @@ def load_spec(files, iextensions=None, extract='opt', flux=True):
     # Load spectra
     spectra_list = []
     for ii,fname in enumerate(files):
-        #msgs.info("Loading extension {:d} of spectrum {:s}".format(extensions[ii], fname))
+        msgs.info("Loading extension {:d} of spectrum {:s}".format(extensions[ii], fname))
         spectrum = arload.load_1dspec(fname, exten=extensions[ii], extract=extract, flux=flux)
+        # Polish a bit -- Deal with NAN, inf, and *very* large values that will exceed
+        #   the floating point precision of float32 for var which is sig**2 (i.e. 1e38)
+        bad_flux = np.any([np.isnan(spectrum.flux), np.isinf(spectrum.flux),
+                           np.abs(spectrum.flux) > 1e30,
+                           spectrum.sig**2 > 1e10,
+                           ], axis=0)
+        if np.sum(bad_flux):
+            msgs.warn("There are some bad flux values in this spectrum.  Will zero them out (not ideal)")
```

Comment:
When we say "zero them out", we actually mean mask, right? If so, it might be good to revise this message a little